packages/autocannon-ui-frontend/src/components/App.jsx

```diff
@@ -1,17 +1,12 @@
 import React, { useState, useCallback } from 'react'
 
-import {
-  createTheme,
-  makeStyles,
-  ThemeProvider
-} from '@material-ui/core/styles'
-
-import { CssBaseline, Container, Grid, Button } from '@material-ui/core'
 import '@fontsource/roboto'
 
 import RunOptionsForm from './RunOptionsForm'
 import ResultSet from './ResultSet'
 import CompareDialog from './CompareDialog'
+import { createTheme, ThemeProvider, styled } from '@mui/material/styles'
+import { Button, Container, CssBaseline, Grid } from '@mui/material'
```

Comment:
minor: for consistency, keep the 3rd party imports above the local ones

Reply:
Done